README.md

```diff
@@ -127,15 +127,22 @@ The list of Contributors is automatically generated.
             <sub style="font-size:14px"><b>Jonathan Carroll</b></sub>
         </a>
     </td>
+    <td align="center" style="word-wrap: break-word; width: 150.0; height: 150.0">
+        <a href=https://github.com/Akaame>
+            <img src=https://avatars.githubusercontent.com/u/11539091?v=4 width="100;"  style="border-radius:50%;align-items:center;justify-content:center;overflow:hidden;padding-top:10px" alt=Sıddık AÇIL/>
+            <br />
+            <sub style="font-size:14px"><b>Sıddık AÇIL</b></sub>
+        </a>
+    </td>
+</tr>
+<tr>
```

Comment:
That is not needed. Once it is merged into master the CI will automatically create this and add it to the `README.md`.

See: https://github.com/niklas-heer/speed-comparison/blob/master/.github/workflows/contributors.yml